fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceDAOImpl.java

```diff
@@ -223,18 +234,19 @@ protected Resource createDTO(ResultSet resultSet) throws FHIRPersistenceDataAcce
                 }
             } else {
                 stmtString = String.format(SQL_HISTORY, resourceType, resourceType);
-                if (this.isDb2Database()) {
+                if (isDb2Database()) {
                     stmtString = stmtString + DB2_PAGINATION_PARMS;
                     resources = this.runQuery(stmtString, logicalId, maxResults, offset);
                 } else {
                     stmtString = stmtString + DERBY_PAGINATION_PARMS;
                     resources = this.runQuery(stmtString, logicalId, offset, maxResults);
                 }
             }
-        } catch (SQLException e) {
-            FHIRPersistenceDataAccessException fx = new FHIRPersistenceDataAccessException("Failure running history query");
-            String errMsg = "Failure running history query: " + stmtString;
-            throw severe(log, fx, errMsg, e);
+//        } 
+//        catch (SQLException e) {
+//            FHIRPersistenceDataAccessException fx = new FHIRPersistenceDataAccessException("Failure running history query");
+//            String errMsg = "Failure running history query: " + stmtString;
+//            throw severe(log, fx, errMsg, e);
```

Comment:
What does this mean? Where is the error thrown now>?

Reply:
This should be removed. The block no longer throws SQLException, so this particular catch is no longer required.